docs/kyma/docs/040-cr-installation.md

```diff
@@ -37,3 +37,23 @@ This table lists all the possible parameters of a given resource together with t
 | **metadata.finalizers** | **NO** | Protects the CR from deletion. Read [this](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers) Kubernetes document to learn more about finalizers. |
 | **spec.version** | **NO** | When manually installing Kyma on a cluster, specify any valid [SemVer](https://semver.org/) notation string. |
 | **spec.url** | **YES** | Specifies the location of the Kyma sources `tar.gz` package. For example, for the `master` branch of Kyma, the address is `https://github.com/kyma-project/kyma/archive/master.tar.gz` |
+
+## Install selected components only
+
+This tool installs components specified in `Installation` CR. See the [installer-cr.yaml.tpl](../../../installation/resources/installer-cr.yaml.tpl) file for more details. 
+
```

Comment:
As for the link, currently, internal links do not work on Console UI and on the website so you can either remove it (and rephrase the description) or put the example of a file in the `assets` folder under the `kyma` directory in docs then the link to it would work but not sure how the example would be maintained later on if its definition changes?